apps/authentication/middleware.py

```diff
@@ -35,7 +35,7 @@ def __call__(self, request):
 
         # 这个是 mfa 登录页需要的请求, 也得放出来, 用户其实已经在 CAS/OIDC 中完成登录了
         white_urls = [
-            'login/mfa', 'mfa/select', 'jsi18n/', '/static/',
+            'login/mfa', 'mfa/select', 'mfa/face','jsi18n/', '/static/',
             '/profile/otp', '/logout/',
         ]
         for url in white_urls:
```

Comment:
The above code appears to be from an unspecified application. It seems there's a typo: in the first set of parentheses, one element (white_urls) is missing at position 0. Also note that "login/mfa" should replace the original "login." Please double check your data for accuracy before proceeding with further analysis or coding changes.